20180224_kubernetes/index.html

```diff
@@ -0,0 +1,206 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=1024" />
+    <meta name="apple-mobile-web-app-capable" content="yes" />
+    <title>Introduction to Kubernetes | by Adrien Lemaire @Fandekasp</title>
+
+    <meta name="description" content="Kubernetes presentation on 2018/02/24 at OWDDM" />
+    <meta name="author" content="Adrien Lemaire" />
+
+    <link href="http://fonts.googleapis.com/css?family=Open+Sans:regular,semibold,italic,italicsemibold|PT+Sans:400,700,400italic,700italic|PT+Serif:400,700,400italic,700italic" rel="stylesheet" />
+    <link href="../impress.js/css/impress-demo.css" rel="stylesheet" />
+    <link href="kubernetes.css" rel="stylesheet" />
+</head>
+<body class="impress-not-supported">
+
+<div class="fallback-message">
+    <p>Your browser <b>doesn't support the features required</b> by impress.js, so you are presented with a simplified version of this presentation.</p>
+    <p>For the best experience please use the latest <b>Chrome</b>, <b>Safari</b> or <b>Firefox</b> browser.</p>
+</div>
+
+<div id="impress">
+
+    <div id="history_1" class="step slide" data-x="-1000" data-y="-1500">
+        <q>A very long time ago... <br />Expert system administrators used cartridges, cassettes tapes, floppy disks and optical media</q>
+        <span class="notes">
+            Before the internet, people had to distribute software across cartridges,
+            then cassette tapes, then floppy disks, and optical media.
+        </span>
+    </div>
+
+    <div id="history_2" class="step slide" data-x="0" data-y="-1500">
+        <q>Then came the advent of <strong>Internet</strong></q>
+        <span class="notes">
+            With the advent of internet, we could remotely update software, and
+            devops were making their own scripts to automate parts of the process.
+        </span>
+    </div>
+
+    <div id="history_3" class="step slide text_align" data-x="1000" data-y="-1500">
+        <p>The next logical steps were PaaS and IaaS!</p>
```

Comment:
PaaS and IaaS in one are very abrupt jump. Maybe split into two slides?

Reply:
The talk is focused on Kubernetes, so I was worried to spend too much time in that first "history" part. Also, considering that today's most popular services are offering both PaaS and IaaS services, I was just gonna mention the difference noted in l.64~67.
If you still think I should split, I will :)

Reply:
I think you should split because when I read it, I struggled hard to remember what PaaS and IaaS is ... don't use those abbr. regularly. If I struggle, others will too!